.github/workflows/e2eTest.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           cache-read-only: false
 
       - name: Start Docker Solution
-        run: ./gradlew dockerSolution:ComposeUp -x test --scan --info --stacktrace
+        run: ../gradlew dockerSolution:ComposeUp -x test --scan --info --stacktrace
         working-directory: TrafficCapture
```

Comment:
I'd rather delete this line than use `..` above

Reply:
duplicated in a couple of other places as well

Reply:
...now I'm curious what the difference is!

Reply:
Looked into it, not sure there is a practical difference here

Reply:
Note, I'm happy to revisit this post merge myself if I've got other ideas